cmd/containerboot/usermetrics.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build linux
+
+package main
+
+import (
+	"io"
+	"log"
+	"net"
+	"net/http"
+
+	"tailscale.com/client/tailscale"
+	"tailscale.com/client/tailscale/apitype"
+	"tailscale.com/util/httpm"
+)
+
+// metrics is a simple metrics HTTP server, if enabled it forwards requests to
+// the tailscaled's LocalAPI usermetrics endpoint at /localapi/v0/usermetrics.
+type metrics struct {
+	lc *tailscale.LocalClient
+}
```

Comment:
It feels like it would be cleaner if we could expose `/healthz`, `/metrics` and any other similar unauthenticated endpoints that might get added in future (?) on the same port rather than having users pick a new port for each and run a new http server.
We could have some kind of `LOCAL_ADDRPORT` env var that gets defaulted to something like `0.0.0.0:9002` and something like `ENABLE_METRICS`, `ENABLE_HEALTHZ` etc env vars + some backwards compat code that looks for the old env var fo the health endpoint.
I don't feel very strongly about this though.

Reply:
Agreed also. This PR is already pretty large, with lots of backwards compat concerns to consider, so I think it's probably best if I make a follow-up PR for this.

Reply:
I chose `TS_OBERVABILITY_ADDR_PORT` for the env var that will serve both health and metrics. Other options I considered were `TS_MONITORING_ADDR_PORT`, `TS_LOCAL_ADDR_PORT`, and `TS_STATUS_ADDR_PORT`. LMK if you have thoughts on the naming :)

Reply:
> TS_LOCAL_ADDR_PORT

I would prefer this

I think `TS_OBERVABILITY_ADDR_PORT` is not intuitive for a name for health endpoint port and maybe even more non-intuitive for anything else we might want to serve on that port in the future - when I read the PR description I actually assumed that you didn't put the health endpoint under this port because of the name

Reply:
Thanks for the feedback 👍 I was attempting to make it a bit more intuitive, but sounds like I didn't; local works for me.